misc/testerina/modules/testerina-runtime/src/main/java/org/ballerinalang/test/runtime/entity/CoverageReport.java

```diff
@@ -353,7 +353,8 @@ private void createReport(final IBundleCoverage bundleCoverage, Map<String, Modu
                     }
                     // If file is a source bal file
                     if (sourceFileCoverage.getName().contains(BLANG_SRC_FILE_SUFFIX) &&
-                            !sourceFileCoverage.getName().contains("tests/")) {
+                            !sourceFileCoverage.getName().contains("tests/") &&
+                            !sourceFileCoverage.getName().contains("test_execute-generated_")) {
```

Comment:
We should check why generated sources are not getting added to the exclusion list